modules/terms-and-conditions/modules/terms-and-conditions/options.js

```diff
@@ -0,0 +1,35 @@
+import { StyleSheet } from "react-native";
+
+//Add your Backend's URL here. 
+const url = "https://toc-1-29797.botics.co/modules/terms-and-conditions/";
```

Comment:
I don't think this should be hardcoded to a demo app since that app will not stay up.

I also found a major security issue with that app. Anyone can POST new data without being logged in. That should not be possible. That is part of this module right?

Reply:
Ah, I had that as a dummy value but missed it changing things back from my demo install -- good catch.  I'll update it to use the global options url and path. 

Also great catch on the security issue. That is part of this module. I think I might as well fix it now, since this module is fairly similar to the Privacy Policy module as well. I figure it's a matter of configuring the DRF viewsets to require authentication, but am no security expert so let me know if you have any other pointers.